ui/desktop/electron-app/package.json

```diff
@@ -40,10 +40,11 @@
     "@electron-forge/maker-dmg": "^7.2.0",
     "@electron-forge/maker-squirrel": "^7.2.0",
     "@electron-forge/maker-zip": "^7.2.0",
-    "electron": "26.2.4",
+    "electron": "28.3.3",
     "unzipper": "^0.11.5"
   },
   "resolutions": {
-    "highlight.js": "^10.4.1"
+    "highlight.js": "^10.4.1",
+    "node-abi": "3.64.0"
```

Comment:
You shouldn't need a resolution here it looks like. Our `node-abi` versions can be brought up manually to 3.64.0 in the `yarn.lock` file without a resolution.

Reply:
Right we have chatted about this before a while back. I will update this.

Reply:
Updated. Let me know if I updated the correct parts.